src/screens/ExamplesScreen.tsx

```diff
@@ -1,19 +1,34 @@
 import { useNavigation } from '@react-navigation/native'
 import React, { FC, useCallback } from 'react'
-import { ScrollView } from 'react-native'
+import { ScrollView, Text } from 'react-native'
 
 import { Button } from '~components'
-import { useTheme } from '~hooks'
+import { useColorScheme, useTheme } from '~hooks'
+import { colorSchemesList } from '~providers'
```

Comment:
`colorSchemesList` nie będzie częścią providera tylko constów jak już, ale imo to jest do zdzwonki i refactoru